docs/guide/apm-doc-directory.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[doc-directory]]
+== Documentation directory
+
+The following sections are designed to guide you through the available Elastic APM documentation.
+
+[float]
+=== APM Overview
+
+You're here!
+The APM Overview is your launchpad for getting started with APM.
+We recommend reading all sections in this book to gain a deeper understanding of the Elastic APM ecosystem.
```

Comment:
`book` isn't a bit a technical term here? readers might be overwhelmed if they think that APM docs are like a book...